modules/configure_project/outputs.tf

```diff
@@ -18,7 +18,7 @@ output "watsonx_project_url" {
   description = "The URL of the watsonx project that's created."
 }
 
-output "watsonx_machine_learning_account_id" {
+output "watsonx_ai_runtime_account_id" {
   value       = local.account_id
   description = "The account id of the Watson Machine Learning instance."
```

Comment:
Description should consider new name as mentioned in the documentation - `The account id of the watsonx.ai Runtime instance`

Reply:
Done the changes.